pure-pso/templates/plugin/snapshotclass.yaml

```diff
@@ -0,0 +1,11 @@
+# The snapshotclass will be applied to ose on install
+{{- if eq .Values.orchestrator.name "openshift" }}
```

Comment:
Not sure this is the correct way to do this.
Why are we limiting the install of the snapshot storage class to only OpenShift. Other deployments could install the CRDs by default as well - eg Anthos 1.6 will do it. 
We should check to see if the CRDs are present and then install the snapshot storage class. Setting as the default class should always be false with an option in the `values.yaml` to make it true.

Reply:
So if it's not an OSE, the yaml file will be empty and applying it does nothing?

Reply:
Maybe using the `Capabilities.APIVersions.Has` is a better option:

```
{{ if .Capabilities.APIVersions.Has "snapshot.storage.k8s.io/v1beta1" }}
```

That way the snapshotclass is created if the API resource exists for any platform.

Reply:
@dnix101 thank you and great idea! were just tyring to figure out how to do that after chatting with the team.